ProjectLighthouse.Servers.Website/Controllers/Admin/AdminUserController.cs

```diff
@@ -91,6 +94,57 @@ public AdminUserController(DatabaseContext database)
 
         return this.Redirect($"/user/{targetedUser.UserId}");
     }
+    
+    /// <summary>
+    /// Deletes every comment by the user. Useful in case of mass spam
+    /// </summary>
+    [HttpGet("wipeComments")]
+    public async Task<IActionResult> WipeComments([FromRoute] int id)
+    {
+        UserEntity? user = this.database.UserFromWebRequest(this.Request);
+        if (user == null || !user.IsModerator) return this.NotFound();
+
+        UserEntity? targetedUser = await this.database.Users.FirstOrDefaultAsync(u => u.UserId == id);
+        if (targetedUser == null) return this.NotFound();
+
+        // Find every comment by the user, then set the deletion info on them
+        await this.database.Comments.Where(c => c.PosterUserId == targetedUser.UserId)
+            .ExecuteUpdateAsync(s =>
+                s.SetProperty(c => c.Deleted, true)
+                 .SetProperty(c => c.DeletedBy, user.Username)
+                 .SetProperty(c => c.DeletedType, "moderator"));
+        Logger.Success($"Deleted comments for {targetedUser.Username} (id:{targetedUser.UserId})", LogArea.Admin);
+
+        await this.database.SendNotification(targetedUser.UserId,
+            "Your comments have been deleted by a moderator.");
+
+        await this.database.SaveChangesAsync();
```

Comment:
```suggestion
```
ExecuteDelete and ExecuteUpdate don’t use change tracking so there’s no need to call SaveChanges